### DISCUSSION TITLE
Between

### DISCUSSION THREAD

**Discussion by jonasgroendahl** [prisma/prisma#14792] [Q&A] [answered]
Hi!

Can you do a query like in Prisma?

```
SELECT name, startsAt, endsAt 
FROM Meeting 
WHERE '2022-08-15' BETWEEN startsAt AND endsAt
```

Only seen greater than, less than etc. operators used for a specific property in a where clause

**Comment by jonasgroendahl** [comment, marked as answer]
I just found this: https://github.com/prisma/prisma/discussions/2045

seems to be emulating same behaviour

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
